recipes/opensim/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+
+# TODO: Dependencies should be separate packages...
+mkdir opensim_dependencies_build
+cd opensim_dependencies_build
+cmake ../dependencies/ \
+      -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+      -DCMAKE_BUILD_TYPE=Release
+make -j8
+cd ..
+
+cp -r $PREFIX/BTK/lib/btk-0.4dev/* $PREFIX/lib/
+cp -r $PREFIX/simbody/libexec/simbody/* $PREFIX/bin/
```

Comment:
I think you want to still put this in the `libexec/simbody` folder; otherwise, I don't think OpenSim will find the `simbody-visualizer`.

If you use the `-DOPENSIM_COPY_DEPENDENCIES=on` flag below, you should not need to manually copy libraries for BTK or Simbody.